<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_NewKeyStore</class>
 <widget class="QDialog" name="Dialog_NewKeyStore">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>426</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>470</width>
    <height>426</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>470</width>
    <height>426</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>New Key Store</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>76</y>
     <width>401</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>76</y>
     <width>18</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Key</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>90</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Key store path:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>48</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>263</x>
     <y>45</y>
     <width>48</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>156</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>164</y>
     <width>96</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Validity(years):</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>186</y>
     <width>449</width>
     <height>189</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>449</width>
     <height>189</height>
    </size>
   </property>
   <property name="title">
    <string>Certificate</string>
   </property>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>20</y>
      <width>120</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>First and Last Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>47</y>
      <width>120</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Organizational Unit:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>74</y>
      <width>78</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Organization:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>100</y>
      <width>102</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>City or Locality:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>127</y>
      <width>108</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>State or Province:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>154</y>
      <width>108</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Country Code (XX):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Name">
    <property name="geometry">
     <rect>
      <x>158</x>
      <y>20</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>281</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>281</width>
      <height>20</height>
     </size>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;名称(CN)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>arno</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_OrganizationalUnit">
    <property name="geometry">
     <rect>
      <x>158</x>
      <y>50</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>281</width>
      <height>20</height>
     </size>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;组织单位(OU)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Organization">
    <property name="geometry">
     <rect>
      <x>158</x>
      <y>76</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>281</width>
      <height>20</height>
     </size>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;组织(O)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Locality">
    <property name="geometry">
     <rect>
      <x>158</x>
      <y>102</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>281</width>
      <height>20</height>
     </size>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;省市(L)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_State">
    <property name="geometry">
     <rect>
      <x>158</x>
      <y>128</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>281</width>
      <height>20</height>
     </size>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;市或者洲(ST)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_CountryCode">
    <property name="geometry">
     <rect>
      <x>158</x>
      <y>154</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>281</width>
      <height>20</height>
     </size>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;国家代码(C)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>86</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="lineEdit_KeyStorePath">
   <property name="geometry">
    <rect>
     <x>123</x>
     <y>16</y>
     <width>297</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>297</width>
     <height>20</height>
    </size>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;keystore.jks存储位置!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>keystore.jks</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_StorePass">
   <property name="geometry">
    <rect>
     <x>123</x>
     <y>45</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>133</width>
     <height>20</height>
    </size>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;默认:arno123&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>arno123</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_StorePassConfirm">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>45</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>133</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>arno123</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_OpenKeystore">
   <property name="geometry">
    <rect>
     <x>423</x>
     <y>16</y>
     <width>25</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>25</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>25</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>104</y>
     <width>36</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Alias:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Alias">
   <property name="geometry">
    <rect>
     <x>123</x>
     <y>100</y>
     <width>325</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>325</width>
     <height>20</height>
    </size>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;别名&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>arno</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_KeyPassConfirm">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>128</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>133</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>arno123</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_KeyPass">
   <property name="geometry">
    <rect>
     <x>123</x>
     <y>128</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>133</width>
     <height>20</height>
    </size>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;默认:arno123&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>arno123</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>132</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>263</x>
     <y>128</y>
     <width>48</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_Validity">
   <property name="geometry">
    <rect>
     <x>123</x>
     <y>160</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;有效期&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="value">
    <number>25</number>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>388</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_KeyStorePath</tabstop>
  <tabstop>pushButton_OpenKeystore</tabstop>
  <tabstop>lineEdit_StorePass</tabstop>
  <tabstop>lineEdit_StorePassConfirm</tabstop>
  <tabstop>lineEdit_Alias</tabstop>
  <tabstop>lineEdit_KeyPass</tabstop>
  <tabstop>lineEdit_KeyPassConfirm</tabstop>
  <tabstop>spinBox_Validity</tabstop>
  <tabstop>lineEdit_Name</tabstop>
  <tabstop>lineEdit_OrganizationalUnit</tabstop>
  <tabstop>lineEdit_Organization</tabstop>
  <tabstop>lineEdit_Locality</tabstop>
  <tabstop>lineEdit_State</tabstop>
  <tabstop>lineEdit_CountryCode</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
